### PR TITLE
PR_2

### PR DESCRIPTION
**Pull request by OksanaEfanova**
Just test PR

### CODE REVIEW

**Comment by alexyer** [comment]
test

File: test.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+line1
```
Comment (tolyandrov) [inline test.py]: review

File: test.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+line2
```
Comment (tolyandrov) [inline test.py]: APPROVED

File: test.py (hunk 1 above)
Comment (OksanaEfanova) [inline test.py]: comment

Comment (OksanaEfanova) [inline test.py]: comment2

**Comment by OksanaEfanova** [comment]
comment

File: test.py (hunk 2 above)
Comment (OksanaEfanova) [inline test.py]: comment2

**Comment by alexyer** [comment]
Hey there

**Comment by alexyer** [comment]
Test

File: test.py (hunk 2 above)
Comment (alexyer) [inline test.py]: test

File: test.py (hunk 1 above)
Comment (OksanaEfanova) [inline test.py]: comment3

**Comment by OksanaEfanova** [comment]
testcomment

**Comment by alexyer** [comment]
test

**Comment by alexyer** [comment]
test

File: test.py (hunk 1 above)
Comment (OksanaEfanova) [inline test.py]: comment4

**Comment by OksanaEfanova** [comment]
comment5

**Comment by OksanaEfanova** [comment]
comment6

File: test.py (hunk 2 above)
Comment (OksanaEfanova) [inline test.py]: test2

File: test.py (hunk 1 above)
Comment (OksanaEfanova) [inline test.py]: comment5

File: test.py (hunk 2 above)
Comment (alexyer) [inline test.py]: test

**Pull request closed without merging**